src/web/client/schema/portalSchema.ts

```diff
@@ -307,6 +308,7 @@ export const portal_schema_V2 = {
                     ["content.customjavascript", "customjs.js"],
                     ["content.copy", "webpage.copy.html"],
                 ]),
+                _content: "content",
```

Comment:
we are already getting content in call above - thats how we are fetching content.copy
DO not add this param here